### ISSUE TITLE
Idea: Error message to FAQ page linking via regexps

### ISSUE THREAD

**Issue by blackspherefollower** [intiface/intiface-central#165]
I was wondering if it'd be not too hard to have an external JSON file loaded into Intiface that basically maps error messages via regexp to FAQ links - this way, common errors could link to their explanations/troubleshooting guides (including new ones as they start appearing) without needing to update Intiface when new cases are added to the FAQ.

**Comment by qdot** [comment]
Yup, that'd be fantastic. The melonloader discord server does a version of this, where people can upload logs from their modded games and get automated help.
